### DISCUSSION TITLE
Order rate-limited dashboard items as queue instead of stack

### DISCUSSION THREAD

**Discussion by katexochen** [renovatebot/renovate#23893] [Suggest an Idea]
### Tell us more.

Currently, upgrades that are listed in the dependency dashboard under "Rate-limited" are managed as a stack: Renovate always opens the item at the top of the list first, and new updates are also added at the top of the list. On bigger repositories, dependencies with frequent updates will get updated all the time, while others are at the bottom of the list and will never be automatically opened.

Consider implementing the list as a queue, so new updates are enqueued at the bottom and renovate opens PRs for the dependencies at the top.

**Comment by rarkins** [comment]
Your description of how they're ordered is not correct. It's not "last in first out".

Patch are favored over minor, minor over major, etc.

The config option `prPriority` can also be used to override

**Reply by katexochen** [reply]
Within a priority group, it is "last in first out". Grouping by priority doesn't change that problem at all.
To which value should I set `prPriority` to prioritize new updates?

**Reply by rarkins** [reply]
If all patch or minor updates are sorted by release date, I don't recall that being intentional and I have a lot of doubts that that's really the case. And in fact I would be happy for that sorting to be reversed if so, so that updates within the same priority are sorted by oldest first.
